CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - Supports fauna database API v5
 - Implemented new driver exception hierarchy
+- Contains, RefClass, Class, ScopedClass, Classes, ScopedClasses, NextID, CreateClass
+  were deprecated.
```

Comment:
"Deprecated" means "not longer recommended for use", not "removed".

We have not yet removed support for the v3 API (or earlier) functions from the core database. See: https://docs.fauna.com/fauna/current/api/fql/deprecations

If we intend to remove the functions from the driver, instead of marking them as deprecated, we should amend this line to read:

```suggestion
- The deprecated functions Contains, RefClass, Class, ScopedClass, Classes,
  ScopedClasses, NextID, CreateClass were removed.
```